proposals/acp/solid-acp.ttl

```diff
@@ -100,6 +100,8 @@ acp:accessControl a rdf:Property ;
     rdfs:label "accessControl"@en ;
     rdfs:comment """The object of this relationship is the access control resource associated with the subject resource.
  This relationship is intended to be used as the rel type of a Link, via HTTP Link Headers rfc5988 for Linked Data resources."""@en ;
+    rdfs:domain gen:InformationResource ;
```

Comment:
can we simply use `ldp:Resource` ?

Reply:
yes. I guess one could. Perhaps gen:InformationResource is too large. I was thinking ldp:Resource would be too narrow. But can enlarge later too.

Reply:
actually `gen:InformationResource` seems ok, I just thought to use something more people may be already familiar with